cypress/utils/plugins/security/constants.js

```diff
@@ -64,3 +65,9 @@ export const SEC_API_CACHE_PURGE_PATH = BASE_SEC_API_PATH + '/cache';
 
 export const SEC_API_AUTHINFO_PATH = BASE_PATH + '/api/v1/auth/authinfo';
 export const SEC_API_TENANT_PATH = BASE_PATH + '/api/v1/multitenancy/tenant';
+
+// STACK MANAGEMENT PATH
+export const STACK_MANAGEMENT_PATH = BASE_PATH + '/app/management';
+
+export const INDEX_PATTERN_PATH = STACK_MANAGEMENT_PATH + '/opensearch-dashboards/indexPatterns';
+export const SAVED_OBJECTS_PATH = STACK_MANAGEMENT_PATH + '/opensearch-dashboards/objects';
```

Comment:
Oh wow, i didnt know we had these constants already defined. We should move these to a `./cypress/utils/dashboards/constants.js` since they are generic to OpenSearch Dashboards. Additionally this is a lil important imo since we export all the constants globally in `./cypress/utils/constants.js`. That might risk us defining the same variable twice if not namespaced correctly.

Reply:
I have moved this into the `./cypress/utils/dashboards/constants.js` with an updated import in my test.